requirements/edx/development.txt

```diff
@@ -42,7 +42,7 @@ git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.1.6#egg=xblock-d
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 git+https://github.com/open-craft/xblock-poll@add89e14558c30f3c8dc7431e5cd6536fff6d941#egg=xblock-poll==1.5.1
 -e git+https://github.com/raccoongang/xblock-pdf.git@sgab-v.1.0.1#egg=xblock-pdf
-git+ssh://git@github.com/edly-io/edly-panel-edx-app.git@edly-1.1.1#egg=edly-panel-app==edly-1.1.1
+git+ssh://git@github.com/lumsx/edly-panel-edx-app.git@develop#egg=edly-panel-app
```

Comment:
1. How will this change behave in the production environment for LUMSx? Are we using the develop branch in production for`edly-panel-edx-app`?
2. I think it's safe to add a tag name or commit hash as the installation version here